docs/ops/upgrading.md

```diff
@@ -35,23 +35,105 @@ under the License.
 
 ## Upgrading the Flink Framework Version
 
-  - Either "in place" : Savepoint -> stop/cancel -> shutdown cluster -> start new version -> start job 
-  - Another cluster variant : Savepoint -> resume in other cluster -> "flip switch" -> shutdown old cluster
+This section describes the general way of upgrading Flink framework version from version 1.1.x to 1.2.x and migrating your
+jobs between the two versions.
+
+In a nutshell, this procedure consists of 2 fundamental steps:
+
+1. Take a savepoint in Flink 1.1.x for the jobs you want to migrate.
+2. Resume your jobs under Flink 1.2.x from the previously taken savepoints.
+
+Besides those two fundamental steps, some additional steps can be required that depend on the way you want to change the
+Flink version. In this guide we differentiate two approaches to upgrade from Flink 1.1.x to 1.2.x: **inplace** upgrade and 
```

Comment:
**in-place** would be better than **inplace** (here and also in the next paragraph)